include/TPP/Dialect/Xsmm/XsmmUtils.h

```diff
@@ -21,6 +26,22 @@ class ArrayAttr;
 class Operation;
 
 namespace xsmm {
+
+struct BrgemmInfo {
```

Comment:
Why do we need the BRGEMM Info struct when lowering Transpose.

Reply:
Agreed, will remove this. It got left out while moving the code.